src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -24,10 +23,23 @@ object InputProcessor {
     val globalOrigin = globalOriginTokens map(_.get)
     val globalOriginArr = Array(globalOrigin(0), globalOrigin(1), globalOrigin(2))
 
-    val remainingLines = lines.tail
-    val transitionIndex = remainingLines.indexWhere(_._1 == "%")
+    val boundingBoxTokens = boundingBoxLine.split(" ") map {x => Try(x.toDouble) }
+    val bbErrIndex = boundingBoxTokens.indexWhere(_.isFailure)
+    if (bbErrIndex != -1) {
+      println(s"Error, Line 2, Token $errIndex: Invalid double found in definition of bounding box")
+      return None
+    } else if (boundingBoxTokens.length !=  6) {
+      println("Error, Line 2: Input file must specify domain as bounding box with 6 double values")
+      return None
+    }
+    val boundingBox = boundingBoxTokens map(_.get)
+    val boundingBoxArr = Array(boundingBox(0), boundingBox(1), boundingBox(2),
+                               boundingBox(3), boundingBox(4), boundingBox(5))
+
+    val remainingLines = lines.tail.tail
```

Comment:
These weird things seem to make sense in the thick of it... When looking at it again it definitely looks weird.